client/my-sites/plugins/plugin-autoupdate-toggle/index.jsx

```diff
@@ -16,12 +16,17 @@ module.exports = React.createClass( {
 	displayName: 'PluginAutopdateToggle',
 
 	propTypes: {
+		isMock: React.PropTypes.bool,
 		site: React.PropTypes.object.isRequired,
 		plugin: React.PropTypes.object.isRequired,
 		wporg: React.PropTypes.bool
 	},
 
 	toggleAutoupdates: function() {
+		if( this.props.isMock ) {
```

Comment:
missing a space in `if(`. I think showing the toggle is important. I wonder if we could capture the click event on the layer above. And not make it something that gets passed to the children under below. I am not sure if this approach would work as well because the user could tab into the toggle.